tests/integration/consumer/test_consumer_memberid.py

```diff
@@ -29,11 +29,12 @@ def test_consumer_memberid(kafka_cluster):
 
     consumer = kafka_cluster.consumer(consumer_conf)
 
+    assert consumer is not None
     kafka_cluster.seed_topic(topic, value_source=[b'memberid'])
 
     consumer.subscribe([topic])
     msg = consumer.poll(10)
     assert msg is not None
     assert msg.value() == b'memberid'
-    assert consumer.memberid() is not None
+    assert len(consumer.memberid()) > 0
```

Comment:
:+1:

Reply:
Also add a type check to verify the returned memberid is indeed the string type.